identityctl/src/main/java/com/android/identity/identityctl/IdentityCtl.kt

```diff
@@ -168,6 +164,74 @@ object IdentityCtl {
         println("- Wrote DS certificate to $certificateOutputFilename")
     }
 
+    fun generateReaderRoot(args: Array<String>) {
+        val certificateOutputFilename =
+            getArg(args,"out_certificate","reader_root_certificate.pem")
+        val privateKeyOutputFilename =
+            getArg(args,"out_private_key","reader_root_private_key.pem")
+
+        val subjectAndIssuer =
+            getArg(args,"subject_and_issuer",
+                "CN=OWF Identity Credential TEST Reader CA,C=UT")
+
+        // From 18013-5 Annex B: 3-5 years is recommended
+        //                       Maximum of 20 years after “Not before” date
```

Comment:
It might be helpful to check this and print out a warning if non-conformant so the user is aware